applications/OptimizationApplication/python_scripts/controls/thickness/shell_thickness_control.py

```diff
@@ -84,58 +36,48 @@ def __init__(self, name: str, model: Kratos.Model, parameters: Kratos.Parameters
 
         default_settings = Kratos.Parameters("""{
             "controlled_model_part_names": [],
-            "filter_settings": {
-                "type" : "implicit",
-                "radius": 0.000000000001,
-                "linear_solver_settings" : {}
-            },
-            "beta_value": 25.0,
-            "penalty_power": 1,
-            "output_all_fields": false,
-            "initial_physical_thickness":0.000001,
-            "physical_thicknesses": [],
-            "fixed_model_parts_and_thicknesses": {},
```

Comment:
why such inputs and implementations are removed !!!
fixed_model_parts_and_thicknesses should be present

Reply:
will be done :)